charts/newrelic-k8s-metrics-adapter/values.yaml

```diff
@@ -0,0 +1,89 @@
+# IMPORTANT: The Kubernetes cluster name
+# https://docs.newrelic.com/docs/kubernetes-monitoring-integration
+#
+# licenseKey:
+# cluster:
+# fargate:
+# IMPORTANT: the previous values can also be set as global so that they
+# can be shared by other newrelic product's charts.
+#
+# global:
+#   licenseKey:
+#   cluster:
+#   fargate
+
+personalAPIKey:
+
+accountID:
```

Comment:
isn't better to include all config maps options under config even if we access them one by one?

Reply:
Right, we can wrap it with `config: {}`, I don't mind.